### DISCUSSION TITLE
Brightness Controls Update Same Monitor

### DISCUSSION THREAD

**Discussion by troywein** [MonitorControl/MonitorControl#1639] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

The application correctly recognized both attached LG displays but both the brightness controls update the same display.

### Steps to reproduce

!. change brightness on both monitors....only one display is updated.

### Expected behavior

I would assume that the controls for each monitor would update that respective monitor.

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.4
- Mac model: MacBook Pro M1 Max - Clamshell Mode
- MonitorControl version: 4.2.0
- Monitor(s): 2x LG Ultrafine
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by waydabber** [comment]
This might be an issue related to display identification. This is a complex issue and won't address it in MonitorControl 4. Recommend switching to BetterDisplay where a number of display identification methods are available to configure which might help with multiple identical display issues.
